nb2workflow/nbadapter.py

```diff
@@ -145,7 +145,9 @@ def cast_parameter(x,par):
         if par.get('is_optional', False):
             return None
         else:
-            raise ValueError(f'Non-optional parameter is set to None')
+            raise ValueError(f'Non-optional parameter {par["name"]} is set to None')
+    if x == '' and par.get('is_file_reference', False):
```

Comment:
so, also an empty URL would be not valid ? perhaps only if not,optional?

also,  it would be good to put comment on why the empty string is not a valid value for a filereference

Reply:
Do you propose to add comment in the code? Maybe describing in the documentation will be enough?
Anyway, see comment below.

Reply:
both: in the code, what you wrote in your comment actually is good, then, it has to be mentioned also in the documentation